tests/WatchTests.cs

```diff
@@ -281,14 +266,11 @@ public void WatchServerDisconnect()
         {
             Watcher<V1Pod> watcher;
             Exception exceptionCatched = null;
+            var exceptionReceived = new CountdownEvent(1);
 
             using (var server = new MockKubeApiServer(TestOutput, async httpContext =>
             {
                 await WriteStreamLine(httpContext, MockKubeApiServer.MockPodResponse);
-
-                // make sure watch success
-                await Task.Delay(TimeSpan.FromMilliseconds(200));
```

Comment:
remove this delay would cause watch failed
exceptions in https://travis-ci.org/kubernetes-client/csharp/builds/359210626?utm_source=github_status&utm_medium=notification

Reply:
We shouldn't need delays. I will fix the code to make sure that this can work...